CHANGELOG.rst

```diff
@@ -3,46 +3,58 @@ Changelog
 
 in development
 --------------
+
+Major Changes:
+~~~~~~~~~~~~~~
+
+* Fix ``st2ctl reload`` command so it preserves exit code from `st2-register-content` script and
```

Comment:
As you said: 
> "Major Changes" - anything that we want to highlight to users. They don't have to necessarily be huge code changes, but they could be anything of interest to users.

It's a bit confusing and counter-intuitive for me. Eg. these changes are Minor and we call them "Major".
Maybe any other ideas how to name this section?